pkgs/by-name/pi/pipenv-poetry-migrate/package.nix

```diff
@@ -18,16 +18,18 @@ python3Packages.buildPythonApplication rec {
 
   build-system = [ python3Packages.poetry-core ];
 
-  pythonRelaxDeps = [
-    "typer"
-  ];
-
   dependencies = with python3Packages; [
     setuptools # for pkg_resources
     tomlkit
     typer
   ];
 
+  # typer for Click >= 8.2 removed "mix_stderr", upstream pins to 8.1.8
```

Comment:
I think we should instead skip the test - and re-enable it when upstream adopts newer `click`. Otherwise when they finally fix it, the package will fail to build due to `--replace-fail`.

```
  disabledTestPaths = [ "tests/test_cli.py" ];
```